src/pages/manager/CustomersListPage.js

```diff
@@ -3,9 +3,132 @@ const { expect } = require('@playwright/test');
+    this.headerThirdCell = this.tableHeader.getByRole('cell').nth(2); // Post Code
+    this.headerFourthCell = this.tableHeader.getByRole('cell').nth(3); // Account Number
+    this.headerFifthCell = this.tableHeader.getByRole('cell').nth(4); // Delete Button
+
+    //Last Data Row
+    this.lastRow = page.getByRole('row').last();
+    this.lastRowFirstNameCell = this.lastRow.getByRole('cell').nth(0);
+    this.lastRowLastNameCell = this.lastRow.getByRole('cell').nth(1);
+    this.lastRowPostCodeCell = this.lastRow.getByRole('cell').nth(2);
+    this.lastRowAccountNumberCell = this.lastRow.getByRole('cell').nth(3);
+    this.lastRowDeleteButton = this.lastRow.getByRole('button', { name: 'Delete' });
+  
+}
 
   async open() {
     await this.page.goto('/angularJs-protractor/BankingProject/#/manager/list');
   }
-}
\ No newline at end of file
+
+  async reload() {
+    await this.page.reload();
+  }
+
+  async waitForOpened() {
+    await this.page.waitForURL('/angularJs-protractor/BankingProject/#/manager/list');
+    }
+
+  // Search Field Methods
+  async searchCustomerByFirstName(firstName) {
+    await this.searchField.fill(firstName);
+  }
+  async searchCustomerByLastName(lastName) {
+    await this.searchField.fill(lastName);
+  }
+  async searchCustomerByPostCode(postCode) {
+    await this.searchField.fill(postCode);
+  }
+
+  async clearSearchField() {
+    await this.searchField.fill('');
+  }
+
+  // Assertions for Search
+  async assertSearchFieldContainsText(text) {
+    await expect(this.searchField).toHaveValue(text);
+  }
+
+  // Header Assertions
+  async assertHeaderIsVisible() {
+    await expect(this.tableHeader).toBeVisible();
+  }
+
+  async assertHeaderFirstCellContainsText(text) {
+    await expect(this.headerFirstCell).toContainText(text);
+  }
+
+  async assertHeaderSecondCellContainsText(text) {
+    await expect(this.headerSecondCell).toContainText(text);
+  }
+
+  async assertHeaderThirdCellContainsText(text) {
+    await expect(this.headerThirdCell).toContainText(text);
+  }
+
+  async assertHeaderFourthCellContainsText(text) {
+    await expect(this.headerFourthCell).toContainText(text);
+  }
+
+  async assertHeaderFifthCellContainsText(text) {
+    await expect(this.headerFifthCell).toContainText(text);
+  }
+
+  // Row Assertions
+  async asserLastRowIsVisible() {
+    await expect(this.lastRow).toBeVisible();
+  }
+
+  async assertLastRowFirstNameContainsText(firstName) {
+    await expect(this.lastRowFirstNameCell).toContainText(firstName);
+  }
+
+
+  async assertLastRowLastNameContainsText(lastName) {
+    await expect(this.lastRowLastNameCell).toContainText(lastName);
+  }
+
+  async assertLastRowPostCodeContainsText(postCode) {
+    await expect(this.lastRowPostCodeCell).toContainText(postCode);
+  }
+
+  async assertLastRowAccountNumberNotContainsText(text) {
+    await expect(this.lastRowAccountNumberCell).toContainText('');
```

Comment:
The method `assertLastRowAccountNumberNotContainsText` seems to have a misleading name. The implementation checks if the account number cell contains an empty string, which is not aligned with the method name. Consider renaming the method to something like `assertLastRowAccountNumberIsEmpty` for clarity.